cloud-api/src/test/java/com/sequenceiq/cloudbreak/cloud/model/TargetGroupPortPairTest.java

```diff
@@ -0,0 +1,42 @@
+package com.sequenceiq.cloudbreak.cloud.model;
+
+import static org.junit.Assert.assertEquals;
+
+import java.util.HashSet;
+import java.util.Set;
+
+import org.junit.Test;
+
+public class TargetGroupPortPairTest {
+
+    private static final int TRAFFIC_PORT = 443;
+
+    private static final int HEALTH_PORT = 8443;
+
+    private TargetGroupPortPair underTest;
```

Comment:
Is this used?

Reply:
Oh gosh. I'm so used to that unit test pattern I just created that object automatically. No, it's not. I'll remove it.